catroid/src/main/java/org/catrobat/catroid/devices/mindstorms/ev3/sensors/EV3Sensor.java

```diff
@@ -352,6 +255,49 @@ public int getPercentValue() {
 		return siValue;
 	}
 
+	private EV3Command buildCommand(int globalVars, EV3CommandByteCode commandCode, EV3CommandOpCode opCode, boolean
```

Comment:
if you could do an intdef or enum for the addTypeAndMode aswell as addReturnValue the whole thing would read a bit more clearly:
EV3Command command = buildCommand(1, null, EV3CommandOpCode.OP_INPUT_READ, true, false);
-> (eg.)
EV3Command command = buildCommand(1, null, EV3CommandOpCode.OP_INPUT_READ, INCLUDE_TYPE_AND_MODE, NO_RETURN_VARIABLE);